zhaquirks/sonoff/snzb06p.py

```diff
@@ -0,0 +1,88 @@
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.general import Basic, Identify, Ota
+from zigpy.zcl.clusters.measurement import OccupancySensing
+from zigpy.zcl.clusters.security import IasZone
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+SONOFF_CLUSTER_FC11_ID = 0xFC11
+SONOFF_CLUSTER_FC57_ID = 0xFC57
+ATTR_SONOFF_ILLUMINATION_STATUS = 0x2001
+
+
+class IlluminationStatus(t.enum8):
+    """Last measureed state of illumination enum."""
+
+    Dark = 0x00
+    Light = 0x01
+
+
+class SonoffFC11Cluster(CustomCluster):
+    """Sonoff manufacture specific cluster that provides illuminance."""
+
+    name = "Sonoff Manufacture Specific Cluster at 0xFC11"
+    cluster_id = SONOFF_CLUSTER_FC11_ID
+    ep_attribute = "sonoff_manufacturer_specific_FC11"
```

Comment:
I'll need to think about the `name` and `ep_attribute` later.
We technically don't need them for this and they're a bit long maybe.